cmd/collector/app/server/zipkin.go

```diff
@@ -85,14 +85,13 @@ func serveZipkin(server *http.Server, listener net.Listener, params *ZipkinServe
 	r := mux.NewRouter()
 	zHandler := zipkin.NewAPIHandler(params.Handler)
 	zHandler.RegisterRoutes(r)
-
-	origins := strings.Split(strings.ReplaceAll(params.AllowedOrigins, " ", ""), ",")
-	headers := strings.Split(strings.ReplaceAll(params.AllowedHeaders, " ", ""), ",")
+	allowedOrigins := strings.Split(strings.ReplaceAll(params.CORSConfig.AllowedOrigins, " ", ""), ",")
+	allowedHeaders := strings.Split(strings.ReplaceAll(params.CORSConfig.AllowedHeaders, " ", ""), ",")
```

Comment:
I think this can be done once in corscfg/flags when parsing from Viper

Reply:
Done!